apps/frontend/src/modules/TFChain/services/entityService.ts

```diff
@@ -24,28 +23,6 @@ export const createEntitySign = (key: KeyringPair, name: string, country: string
     return Buffer.from(signedMessage).toString('hex');
 };
 
-export const getEntity = async (api: any, id: any) => {
```

Comment:
Can we just remove this?
Don't we need it for farmer?

Reply:
Not needed anymore (even in farmers), all calls for entity or now based on function getEntityIDByAccountId()